### ISSUE TITLE
Move Translation / Language packs site into Docs site

### ISSUE THREAD

**Issue by pgorod** [salesagility/SuiteDocs#207]
@horus68 has requested that we move the site he built with help about Language Packs and translations, into our Docs site here.

The content is to be found at
https://sites.google.com/site/translatingsuitecrm

I suggest that the Language Packs help is integrated into existing sections, while the Translation bits can be inserted in the Community guide, expanding the "Translate" page.

**Comment by pgorod** [comment]
@horus68 can you please provide a zip file with all the images in your screenshots? That should accelerate the process of converting all the content into Asciidoc. Thanks!

**Comment by pgorod** [comment]
@cameronblaikie so, there are 4 pages in that site:

1. Translate online
2. Download languages
3. Install a new language
4. Update a language pack

So I guess number 1 should be a new section `Contributing to Translation` under here: 
https://docs.suitecrm.com/community/

And the rest I suggest can go in a new section `Languages` under here:
https://docs.suitecrm.com/admin/installation-guide/

and make sure to put a link to it from here:
https://docs.suitecrm.com/admin/administration-panel/system/#_languages

I'd like to hear what others think about this distribution of content, is it ok?

**Comment by pgorod** [comment]
Note that there is a page under `Contributing to Docs` which is called `Translate`, but that is a different thing.

**Comment by likhobory** [comment]
I think we can replace Translate file content with the new content. The new guidelines file already contains a necessary and more detailed description  about language code before the extension.

**Comment by pgorod** [comment]
Hhmm you are right that the new Guidelines explain that better, so I need to think about what to do to the current `Translate` page.

https://docs.suitecrm.com/community/contributing-to-docs/translate/

But that is about Translating Docs, not the SuiteCRM app itself. (We can start by renaming that page to `Translating Docs`)

So this content from Horus68 is not the same...

**Comment by likhobory** [comment]
>We can start by renaming that page to `Translating Docs`
Agree.  And it may be two different files: `Translating Docs` and `Translating Lanluage pack` (for example).

**Comment by pgorod** [comment]
I already did some of these fixes: changed title to `Translating Documentation`, added link to Guidelines page.

We can do some more work perfecting this arrangement once we have the content from Horus68 available here.

**Comment by cameronblaikie** [comment]
@pgorod The current translate page is for the docs site

**Comment by pgorod** [comment]
Yes, I updated the title of that page yesterday.

The new content should go in different  places as specified here: https://github.com/salesagility/SuiteDocs/issues/207#issuecomment-461036231

**Comment by pgorod** [comment]
@camy I am leaving this issue open because there's still a bit more work needs to be done:
https://github.com/salesagility/SuiteDocs/pull/219#issuecomment-510990921

**Comment by serhiisamko091184** [comment]
Hello, @pgorod 

The issue has been marked as stale because there has been no recent activity. It will be closed if no further activity occurs.
Thanks for your contributions.

Regards,
Serhii

**Comment by pgorod** [comment]
This is not my work, this is something that the maintainers (or anyone else) might want to do. It would help promote translators' work.
